addons-web-sdk/samples/build.sh

```diff
@@ -0,0 +1,9 @@
+for sample in ./*/
```

Comment:
Could get around this using workspace commands and `concurrently` but this is probably fine. The npm version of these commands is also less nice than the yarn version.

Reply:
OH! I didn't really understand workspaces. Thanks for this! I didn't use concurrently but now it's much cleaner -- just calls `npm run build --workspaces`